### DISCUSSION TITLE
Where to find concourse documentation for older version

### DISCUSSION THREAD

**Discussion by prime89-hash** [concourse/concourse#7673] [General]
Hi, 

Could you please provide the link or details of document for concourse older version.

**Comment by taylorsilva** [comment]
Sadly we never got around to versioning our docs :(

That said, they're pretty unchanged even from the 5.0 era. It's mostly been additions so you may just run into stuff that doesn't exist on your older version.

You could check the repo out to an older date and then run `./scripts/build -s 3000` to run the site locally.

**Reply by prime89-hash** [reply]
Thank you @taylorsilva :) , I have installed go on my machine and clone the docs repo.
I tried to execute command given above ./scripts/build -s 3000 however it is showing error as "booklit: command not found" through we it has installed "go install github.com/vito/booklit/cmd/booklit" still it showing that error, any suggestion will be great :)

**Reply by taylorsilva** [reply]
hmm note sure but maybe check you did everything here: https://github.com/concourse/docs/#building-the-docs-locally
Maybe your `${GO_PATH}/bin` is not on your `$PATH`?

**Reply by prime89-hash** [reply]
Hi @taylorsilva , The path has been set already
PATH=/usr/local/go/bin

**Reply by taylorsilva** [reply]
darn! not sure what else it could be then. Might have to dig into the script yourself to see where it's breaking.

**Comment by prime89-hash** [comment]
Hi @taylorsilva , the issue has been resolved we are now able to open the docs using https://localhost:8000, Thank you so much :).

There are 2 more queries regarding to concourse upgrade from 5.8.1 to 7.5.0, 
1- Is there any changes I can check for Vault integration 
2- Is there anything changed or anything we need to take care related to RDS as we are using postgresql 12.5 for current concourse 5.8.1 version.

**Reply by taylorsilva** [reply]
1. Nope, I think the most recent change was to get vault integration working with kv2
2. No changes on the RDS side with postgres. 12.5 will work just fine
